devtools/test_dashboard/devtools.js

```diff
@@ -4,6 +4,7 @@
 
 var Fuse = require('fuse.js');
 var mocks = require('../../build/test_dashboard_mocks.json');
+var credentials = require('../../build/credentials.json');
```

Comment:
So `npm run start` now requires devs to sign up for mapbox and run `npm run pretest`

Maybe we should make this less stringent?